config/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "development": {
```

Comment:
Also, look into how to setup database with [render](https://dashboard.render.com/d/dpg-c07dao5c2rvvqlb66160)

Reply:
Will figure it out later in a different PR.

Reply:
It's important now, Local build for database setup doesn't help.

Reply:
> Will figure it out later in a different PR.

Got it. For now, testing it locally then.

@manangouhari can you remove all dependency related to config lib since we are now using dotenv. And shift all config settings i.e database settings inside .env file.